include/lexy/callback/bind.hpp

```diff
@@ -347,13 +349,24 @@ struct _bound_sink
     LEXY_EMPTY_MEMBER Sink _sink;
     LEXY_EMPTY_MEMBER _detail::tuple<BoundArgs...> _bound;
 
-    /*
-        constexpr auto sink() const
-        {
-            return _detail::invoke_bound(_sink_wrapper<Sink>{_sink}, _bound,
-       _bound.index_sequence(), _detail::no_bind_context{});
-        }
-        */
+    template <typename... Args>
+    constexpr auto operator()(Args... args) const
+    {
+        return _sink(std::forward<Args>(args)...);
+    }
+
+    template <typename... Args>
+    constexpr auto operator()(Args... args)
```

Comment:
This overload is unnecessary: the callbacks are all `constexpr` objects anyway, so everything has to be `const` qualified.

Reply:
doh - of course!